src/main/webapp/i18n/en/calendar.json

```
@@ -0,0 +1,42 @@
+{
+    "artemisApp": {
+        "calendar": {
+            "mondayShort": "Mon",
+            "tuesdayShort": "Tue",
+            "wednesdayShort": "Wed",
+            "thursdayShort": "Thu",
+            "fridayShort": "Fri",
+            "saturdayShort": "Sat",
+            "sundayShort": "Sun",
+            "eventSubtypeName": {
+                "lectureStart": "Start",
+                "lectureEnd": "End",
+                "examPublishResults": "Results Out",
+                "examReviewStart": "Review Start",
+                "examReviewEnd": "Review End",
+                "exerciseRelease": "Release",
+                "exerciseStart": "Start",
+                "exerciseDue": "Due",
+                "exerciseAssessmentDue": "Assessment Due"
+            },
+            "exerciseFilterOption": "Exercise Events",
+            "lectureFilterOption": "Lecture Events",
+            "tutorialFilterOption": "Tutorial Events",
+            "examFilterOption": "Exam Events",
+            "eventName": {
+                "programming": "Programming Exercise Event",
+                "text": "Text Exercise Event",
+                "fileUpload": "File-Upload Exercise Event",
+                "modeling": "Modeling Exercise Event",
+                "quiz": "Quiz Exercise Event",
+                "lecture": "Lecture Event",
+                "tutorial": "Tutorial Event",
+                "exam": "Exam Event"
```

I would suggest to remove the "event(s)" appendices

```suggestion
            "exerciseFilterOption": "Exercises",
            "lectureFilterOption": "Lectures",
            "tutorialFilterOption": "Tutorials",
            "examFilterOption": "Exams",
            "eventName": {
                "programming": "Programming Exercise",
                "text": "Text Exercise",
                "fileUpload": "File-Upload Exercise",
                "modeling": "Modeling Exercise",
                "quiz": "Quiz Exercise",
                "lecture": "Lecture",
                "tutorial": "Tutorial",
                "exam": "Exam"
```